mono/mini/il2cpp-stubs.cpp

```diff
@@ -255,13 +255,13 @@ int il2cpp_mono_string_length (MonoString *monoStr)
 	return str->length;
 }
 
-char* il2cpp_mono_array_addr_with_size (Il2CppMonoArray *array, int size, uintptr_t idx)
+char* il2cpp_mono_array_addr_with_size (MonoArray *array, int size, uintptr_t idx)
```

Comment:
I am not sure we should use Mono types here. Can't we use our types if that is what we are really expecting?

Reply:
I could go either way on that honestly. from the mono side you're passing in "MonoArray" into these functions. I do plan on going through afterwards and making this file consistent. Theres a lot of places where we would cast the Il2CppMonoType as an Il2CppType which are no longer necessary since they're the same type now